maat-court-data-api/src/main/java/gov/uk/courtdata/reporder/controller/CCOutcomeController.java

```diff
@@ -145,4 +150,20 @@ public ResponseEntity<List<RepOrderCCOutcomeDTO>> findByRepIdLengthInHeader(@Pat
         responseHeaders.setContentLength(service.findByRepId(repId).size());
         return ResponseEntity.ok().headers(responseHeaders).build();
     }
+
+    @Operation(description = "Deleting Crown Court Outcome by a RepOrder")
+    @ApiResponse(responseCode = "200", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE))
+    @StandardApiResponse
+    @DeleteMapping(value = "/rep-order/{repId}", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<Integer> deleteCrownCourtOutcome(@PathVariable final Integer repId) {
+        if(nonNull(repId)){
```

Comment:
I don't think you need this null check, as since repId is a path variable it cannot by definition be non-null?